resources/js/data-access/memo-api/tquery/table.ts

```diff
@@ -4,53 +4,44 @@ import {NON_NULLABLE, debouncedFilterTextAccessor} from "components/utils";
 import {Accessor, Signal, createComputed, createMemo, createSignal, on} from "solid-js";
 import {FilterH, FilterReductor} from "./filter_utils";
 import {RequestCreator} from "./tquery";
-import {Column, ColumnName, DataRequest, DataResponse, Schema} from "./types";
+import {Column, ColumnName, DataRequest, DataResponse} from "./types";
+
+export interface ColumnConfig {
```

Comment:
Może niech to będzie obok PartialColumnConfig i FullColumnConfig?

Reply:
To by była zależność cykliczna, wprawdzie można z tym żyć ale to jest próba zrobienia izolacji. Że tu definiuję tylko to, co jest tu potrzebne, a jak tabela będzie te configi produkować i dostarczać to jej sprawa.